.github/workflows/pr_epm.yml

```diff
@@ -141,6 +141,7 @@ jobs:
     - name: Install centos dependency
       if: ${{ contains(matrix.tag, 'centos') }}
       run: docker exec $inclavare_test bash -c 'cd /root/inclavare-containers/${{ matrix.tag }};
+        sleep 20 && systemctl stop dnf-makecache.timer;
```

Comment:
so why waiting for 20 seconds here?

Reply:
Waiting for `systemd` running in container.

Reply:
Add this info to comments as well.